repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -78,29 +92,37 @@ def _import_csv_to_rstuf(
         console.print(f"Import status: {csv_file} imported")
 
 
-def _get_succinct_roles(metadata_url: str) -> SuccinctRoles:
-    response = request_server(metadata_url, "1.bin.json", Methods.get)
-    if response.status_code == 404:
-        raise click.ClickException("RSTUF Metadata Targets not found.")
+def _get_succinct_roles(api_url: str) -> SuccinctRoles:
+    response = request_server(api_url, URL.config.value, Methods.get)
+    if response.status_code != 200:
+        raise click.ClickException(
+            f"Failed to retrieve RSTUF config {response.text}"
+        )
 
-    json_data = json.loads(response.text)
-    targets: Metadata[Targets] = Metadata.from_dict(json_data)
-    if targets.signed.delegations is None:
-        raise click.ClickException("Failed to get Targets Delegations")
+    try:
+        data = response.json()["data"]
+        num_bins = data["number_of_delegated_bins"]
 
-    targets_delegations: Delegations = targets.signed.delegations
+    except (json.JSONDecodeError, KeyError):
+        raise click.ClickException(
+            "Failed to parse 'data', 'number_of_delegated_bins' from config "
+            f"{response.text}"
+        )
+    bit_length = int(log(num_bins, 2))
 
-    if targets_delegations.succinct_roles is None:
-        raise click.ClickException("Failed to get Targets succinct roles")
+    # the 'keyids' and the 'threshold' are irrelevant once we need the names
```

Comment:
I don't understand what you mean by this comment.
Maybe something like this: 
```suggestion
    # 'threshold' is always 1 as we use a single online key 'keyids' will be
    # populated  when the online key is added
```

Reply:
Even if we decide to use multiple online keys or using offline keys for the targets, it is still irrelevant as we don't do operations to the metadata, only the database.